src/parser.js

```diff
@@ -263,6 +263,17 @@ export default class Parser extends Stream {
                 return;
               }
 
+              if (entry.attributes.KEYFORMAT === 'com.microsoft.playready') {
+                this.manifest.contentProtection = this.manifest.contentProtection || {};
+
+                // TODO: add full support for this.
+                this.manifest.contentProtection['com.microsoft.playready'] = {
+                  url: entry.attributes.URI
```

Comment:
Since the attribute is `URI` and we use `uri` in most places within the parser, it may make sense to use `uri` here.

Reply:
seems like we use `url` for `keySystems`

Reply:
Yeah, I think that was my mistake when using both `uri` and `url` within videojs-contrib-eme's options. I wonder if we're better off for parsing within m3u8-parser trying to keep consistent with the naming, and then do translation if we have to in VHS. It might also make sense to rename the `url` option in videojs-contrib-eme to `uri`, or at least make both available.